incubator/hnc/test/e2e/issues_test.go

```diff
@@ -237,3 +238,80 @@ var _ = Describe("Issues", func() {
 		mustRun("kubectl get rolebinding foo -n", nsChild, "-oyaml")
 	})
 })
+
+var _ = Describe("Issues that require reparing HNC", func() {
+
+	hncRecoverPath := os.Getenv("HNC_REPAIR")
```

Comment:
Turn `"HNC_REPAIR"` into a constant that's shared?

Reply:
done